test/factories/coins.rb

```diff
@@ -0,0 +1,54 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :coin do
+    sequence(:name) { |n| "#{Faker::CryptoCoin.coin_name}#{n}" }
```

Comment:
I can't believe `Faker::CryptoCoin` is a thing haha

Reply:
Haha, it's not really that useful though